bin/rucio-replica-recoverer

```diff
@@ -244,7 +242,8 @@ Note that attempting the use the ``--vos`` argument when in single-VO mode will
     parser.add_argument('--vos', nargs='+', type=str, help='Optional list of VOs to consider. Only used in multi-VO mode.')
     parser.add_argument("--run-once", action="store_true", default=False, help='One iteration only.')
     parser.add_argument("--limit-suspicious-files-on-rse", action="store", default=5, help='Maximum number of suspicious replicas on an RSE before that RSE is considered problematic and the suspicious replicas on that RSE are declared "TEMPORARY_UNAVAILABLE". Default value is 5.')
-    parser.add_argument('--sleep-time', action="store", default=300, type=int, help='Concurrency control: thread sleep time after each chunk of work')
+    parser.add_argument('--sleep-time', action="store", default=3600, type=int, help='Concurrency control: thread sleep time after each chunk of work')
+    parser.add_argument('--passive-mode', action="store", default=False, help='Mainly for testing purposes. If True, the daemon will be run without taking any actions on any files. The log file will be produced normally. By default False.')
```

Comment:
- `action="store_true"` is more appropriate here. It sets the value of the variable to `True` if specified, `False` otherwise. If you use `store`, the user has to provide a value: `--passive-mode True`.
- The default case can be omitted then.
- From my POV, the text should be edited to something like: ~Mainly for testing purposes.~ If ~True~ specified, the ... . ~By default False.~
https://docs.python.org/3/library/argparse.html#action

Reply:
I'm going to change this to `--disable-passive-mode`, `action=store_false` and `default=True`, so that the passive mode is run by default. I'm going to kept the argument `passive_mode=ARGS.disable_passive_mode` in `run`, so that we can avoid having to implement a double-negative (something like `disable_passive_mode=False`) in `suspicious_replica_recoverer.py`.